packages/syft/src/syft/service/worker/worker_pool.py

```diff
@@ -62,14 +85,30 @@ def get_job_repr(self):
         else:
             return ""
 
-    def _coll_repr_(self):
-        return {
-            "name": self.name,
-            "container id": self.container_id,
-            "status": str(self.status.value.lower()),
-            "job": self.get_job_repr(),
-            "created at": str(self.created_at),
-        }
+    def get_status_healthcheck(self) -> None:
```

Comment:
Health checks should be done via API call to the serve not on the client side.

Reply:
Thanks @shubham3121, done!